Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -51,6 +51,11 @@ public class PressableButton : MonoBehaviour, IMixedRealityTouchHandler
         [FormerlySerializedAs("returnRate")]
         private float returnSpeed = 25.0f;
 
+        [SerializeField]
+        [Tooltip("Button will send the release event on touch end after successful press even if release plane hasn't been passed.")]
+        private bool releaseOnTouchEnd = true;
+        public bool ReleaseOnTouchEnd { get => releaseOnTouchEnd; set => releaseOnTouchEnd = value; }
```

Comment:
doc comment so people can see in the api docs as well?

Reply:
added comments to the other public fields for pressable button as well